### PR TITLE
PFG migration

### PR DESCRIPTION
**Pull request by thet**
Changes by @buchi 
I'm just opening this PR to not forget about this branch and discuss about it.

@buchi - It looks conceptionally good to me. Did you successfully run a migration from PFG?

### CODE REVIEW

File: src/collective/easyform/configure.zcml [hunk 1]
```diff
@@ -19,6 +19,7 @@
   <include file="upgrades.zcml" />
 
   <include package=".browser" />
+  <include package=".migration" />
```
Comment (thet) [inline src/collective/easyform/configure.zcml:22]:
```suggestion
  <include zcml:condition="installed Products.PloneFormGen" package=".migration" /> 
```

Comment (thet) [reply src/collective/easyform/configure.zcml:22]: Also needs the zcml namespace declaration.

**Comment by pbauer** [comment]
@buchi 🎉 
How come there is no community-post that this exists? I only stumbled over it because @thet operened the PR. We had a short discussion about the requirement for such a migration at the Plone Tagung in Dresden and from about 50 people in the room (among them many core-developers), noone knew that it is already done.

**Comment by laulaz** [comment]
@thet @buchi thanks for the work here !

Some issues, after having used this migration code on one of our customer site :
 - `ploneformgen` should be replaced by `easyform` in `CustomScript` `ScriptBody` field
 - `folder` should be replaced by `here` in TAL expressions
 - PFG `FieldsetFolder` are not migrated

**Comment by laulaz** [comment]
@thet @buchi @pbauer 

I added PFG **`FieldsetFolder`** migration logic that was missing and resolved @thet mini-review 👍.
IMHO this pull request can be merged.

---

For the 2 other (minor ?) issues I found (`ploneformgen` -> `easyform` in scripts & `folder` -> `here` in TAL expressions), this is the **quick and dirty** code I wrote to fix them 😇, of course not included in this PR.

In `actions_model` :
```
            if isinstance(value, str) or isinstance(value, unicode):
                if "ploneformgen" in value:
                    value = value.replace("ploneformgen", "easyform")
```

In `convert_tales_expressions` :
```
    elif 'folder' in value:
        return value.replace("folder", "here")
```

**Comment by thomasmassmann** [comment]
@buchi , is it save to backport this into the 1.x branch? Then we can migrate forms in old 4.x sites without first having to update to 5.x

This would be especially usefull when using collective.exportimport to migrate sites to a new instance, without having the need to upgrade the old site first.

**Pull request merged**